draft/accessibility-team.rst

```diff
@@ -0,0 +1,129 @@
+============================
+DEP XXXX: Accessibility Team
+============================
+
+:DEP: XXXX
+:Author: Tom Carrick
+:Implementation Team: Tom Carrick, others to be determined
+:Shepherd: to be determined
+:Status: Draft
+:Type: Process
+:Created: 2020-06-29
+:Last-Modified: 2020-06-29
+
+.. contents:: Table of Contents
+   :depth: 3
+   :local:
+
+Abstract
+========
+
+This DEP proposes the formation of an Accessibility Team to encourage projects
+maintained by the Django Software Foundation to be accessible to as diverse a
+group as possible, particularly people with disabilities that make using the
+web more difficult.
+
+Specification
+=============
+
+The Accessibility Team (informally, a11y team) shall be formed.
+
+Membership
+----------
+
+The team shall not have a fixed size, but instead will grow and shrink
+organically as members choose to leave, and when new members are deemed to be
+required by the rest of the team. Membership of the team works similarly to the
+`Code of Conduct Committe <https://github.com/django/code-of-conduct/blob/master/membership.md>`_.
+New members shall be chosen from a list of volunteers, or if there is a lack
+of volunteers, an advertisement will be published on the Django website.
+Priority will be given to volunteers who, in no particular order:
+
+- Have disabilities that make using the web and/or web development more
+  difficult.
+- Have relevant, positive contributions or expertise in the field.
+- Have a record of contributing to Django.
+
+Members shall remain in the team for a fixed-term of 9 months, after which
+they must opt-in to remain on the team. Membership will also be terminated by:
+
+- Becoming disqualified due to actions taken by the Code of Conduct committe
+  of the Django Software Foundation.
+
+- A vote of the Technical Board, or full consensus of the rest of the
+  Accessibility Team, if the team is considered too large, the person is not
+  making positive contributions, or any other sound reason.
+  (TODO: dunno about any of this)
+
+Responsibilities
+----------------
+
+The remit of the Accessibility Team shall include all user-facing components
+of projects maintained by the Django Software Foundation. This includes but is
+not limited to:
+
+- User-facing parts of Django, such as default HTML output in forms and such,
+  and the django admin and its default theme, HTML, and JavaScript components.
+
+- All websites and other software projects maintained by the DSF, such as the
+  Django website, including documentation, Django Snippets, Django People,
+  the issue tracker, etc.
+
+The responsibilities of the Accessibility Team are expected to change over
+time, and are to be decided by conensus of the team, with input from the
+Technical Board as required. to begin, several areas have been identified:
+
+- Deciding on any relevant accessibility guidelines to follow, such as WCAG,
+  and at which conformance level.
```

Comment:
I can see why it would be valuable to have that discussion, but at the same time it doesn’t seem to me like there needs to be an accessibility team for this to be decided on – from the get-go there was strong consensus on the Groups thread for WCAG2.0 AA, since lots of national laws are based on it – and from there, WCAG2.1 feel like a natural "might as well" (or 2.2, when that becomes a candidate recommendation).

Reply:
I agree that in principle, this is already decided to some extent. However, I want to leave it in so that the team can decide to follow new standards without e.g. having to go to the mailing list, which for small things like upgrading to a backwards-compatible WCAG version (such as from 2.0 to 2.1) seems unnecessary to me.